src/woo_publications/publications/tests/test_documents_admin.py

```diff
@@ -555,10 +564,19 @@ def test_document_admin_delete(
             model_name="Document",
             uuid=str(document.uuid),
         )
+        mock_remove_document_from_openzaak.assert_called_once_with(
+            document_id=document.id,
+            user_id=self.user.pk,
+            service_uuid=document.document_service.uuid,
+            document_uuid=uuid.UUID(document.document_uuid),  #  pyright: ignore[reportArgumentType]
```

Comment:
I don't know if it's better to mute the typing error here (which shouldn't exist) or cast the `document_uuid` field in the Factory to UUID